helmfile/values/prometheus-blackbox-exporter-wc.yaml.gotmpl

```diff
@@ -66,3 +66,38 @@ serviceMonitor:
       scrapeTimeout: 30s
       module: http_2xx
     {{- end }}
+
+    - name: sc-api-server
+      url: https://kube-apiserver.{{ .Values.global.baseDomain }}:/healthz
```

Comment:
I do not think we are using this subdomain for sc

Reply:
Hm, I think I just copied that from

https://github.com/elastisys/compliantkubernetes-apps/blob/1651f30df8a1d1eb4a98c576cf91a321c06e5d17/helmfile/values/prometheus-blackbox-exporter-sc.yaml.gotmpl#L106

No idea if this is actually relevant but it looks hard-coded here:

https://github.com/elastisys/compliantkubernetes-apps/blob/1651f30df8a1d1eb4a98c576cf91a321c06e5d17/helmfile/charts/kubeapi-metrics/templates/ingress.yaml#L3

Reply:
I need to check and return with an answer

Reply:
this one is used for user-apiserver only, we do not create and ingress for apiserver in sc so I think you can remove this check